zkevm-circuits/src/evm_circuit/execution/pc.rs

```diff
@@ -31,17 +30,17 @@ impl<F: Field> ExecutionGadget<F> for PcGadget<F> {
     const EXECUTION_STATE: ExecutionState = ExecutionState::PC;
 
     fn configure(cb: &mut EVMConstraintBuilder<F>) -> Self {
-        let value = cb.query_word_rlc();
+        let value = cb.query_u64();
 
         // program_counter is limited to 64 bits so we only consider 8 bytes
         cb.require_equal(
             "Constrain program_counter equal to stack value",
-            from_bytes::expr(&value.cells),
+            value.to_word().lo(),
```

Comment:
```suggestion
            value.expr(),
```
I think it's more clear like this.

Reply:
Updated in this commit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1411/commits. And I also check all place which have `value.to_word().lo()` and fixed to `value.expr()` after having `IntDecomposition` type